internal/uow/transaction_test.go

```
@@ -0,0 +1,191 @@
+
+			tx, ok := tt.svc.transactions[actualTx.id]
+			require.True(t, ok)
+			require.NotEmpty(t, tx)
+
+			assert.Equal(t, txStatusInProgress, tx.status)
+			assert.Equal(t, tt.requests, tx.requests)
+		})
+	}
+}
+
+//nolint:funlen,dupl // много тест-кейсов, одинаковые тест-кейсы для разных тестов
```

golangci-lint (2.5.0)

[error] directive `//nolint:funlen,dupl // много тест-кейсов, одинаковые тест-кейсы для разных тестов` is unused for linter "dupl"

(nolintlint)